app/js/components/orderEntry/SearchAndAddOrder.jsx

```diff
@@ -74,11 +74,13 @@ export class SearchAndAddOrder extends React.Component {
   }
 
   handleEditActiveDrugOrder = (order) => {
+    const editOrder = order;
```

Comment:
I don't see how this changes are necessary

Reply:
Looking into it. Thank u

Reply:
When an active order is being passed, the value of the action is `NEW` and this is the same as the draftorder for a new order. The change above is set the active order action to revise so as to get the appropriate action.

Reply:
Got it. Thank you.